rustls-post-quantum/Cargo.toml

```diff
@@ -11,8 +11,12 @@ repository = "https://github.com/rustls/rustls"
 categories = ["network-programming", "cryptography"]
 autobenches = false
 
+[features]
+aws-lc-rs-unstable = ["webpki/aws-lc-rs-unstable"]
+
 [dependencies]
-rustls = { version = "0.24.0-dev.0", features = ["aws-lc-rs", "prefer-post-quantum"], path = "../rustls" }
+rustls = { version = "0.23.29", features = ["aws-lc-rs", "prefer-post-quantum"] }
```

Comment:
Seems like it could make sense for this to stick with 0.23.x for longer?

Reply:
I think this could be made to work -- there's no longer a need for bogo to depend on rustls-post-quantum now it's in the core crate.